remote-config.jsonc

```
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
+                },
+                {
+                    "message": "You can use a more lively `ddev list` and `ddev describe` output with `ddev config global --table-style=bright`. Try it out!"
```

Run commands, drop “out” to save characters with no change in meaning:

```suggestion
                    "message": "You can use a more lively `ddev list` and `ddev describe` output by running `ddev config global --table-style=bright`. Try it!"
```